tests/test_fixtures.py

```diff
@@ -0,0 +1,33 @@
+import pytest
+from pathlib import Path
+import re
+import json
+from jsonschema import (
+    ValidationError as jsonValidationError,
+)
+
+# todo
+# validate manifest file formatting
+# use pytest.fixture to parametrize
```

Comment:
I'd look at using [`pytest_generate_tests`](https://docs.pytest.org/en/latest/parametrize.html#basic-pytest-generate-tests-example).  I believe it's the correct mechanism.  We use it in `py-evm/trinity` for generating tests from the `ethereum/tests` fixtures.

Reply:
ahh great, yeah that did the trick, thanks! this pr is ready for a final :eyes: if you have any other thoughts